### PR TITLE
Moves native routing documentation from openshift-sdn.

### PR DESCRIPTION
**Pull request by mrunalp**
@adellape PTAL.

### CODE REVIEW

**Comment by mrunalp** [comment]
@danmcp PTAL. This PR moves the document as-is (converting md to adoc) from openshift-sdn to openshift-docs.

File: _build_cfg.yml [hunk 1]
```diff
@@ -142,6 +142,8 @@ Topics:
     File: persistent_storage_nfs
   - Name: IPtables
     File: iptables
+  - Name: Native container routing using network infrastructure
```
Comment (adellape) [inline _build_cfg.yml]: I might suggest a shorter title here if possible (maybe "Native Container Routing"?) in the interest of the left nav not wrapping. Note that the title can be different here versus what's in the actual topic title in the .adoc file.

**Comment by adellape** [comment]
@bfallonf FYI per https://trello.com/c/LOjwaEDd.

File: admin_guide/native_routing.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+the kernel networking stack in Linux. The setup requires that the network administrator or some script
+modifies the router[s] as new nodes are added to the cluster. The document will describe the steps to
+use a Linux server as a simple router. The steps could then be adapted to any particular router.
+
+
+== Network Layout
+The diagram below shows the setup used in the document. It has one Linux node with two network interface
+cards serving as a router, two switches and three nodes connected to these switches. 
+
+image::network_diagram.png["Network Diagram"]
```
Comment (adellape) [inline admin_guide/native_routing.adoc]: @mrunalp Sorry, just noticed: the PNG file itself should go in an `admin_guide/images/` folder for it to show up in the build. You can still reference the file here as you are.

Comment (mrunalp) [inline admin_guide/native_routing.adoc]: @adellape Thanks. Fixed.

**Pull request merged**